controller/TaoEventLog.php

```diff
@@ -42,11 +43,25 @@ public function __construct(){
      * A possible entry point to tao
      */
     public function index() {
-        echo __("Hello World");
+        $this->setData('author', 'Open Assessment Technologies SA');
+        $this->setView('TaoEventLog/log.tpl');
     }
 
-    public function templateExample() {
-        $this->setData('author', 'Open Assessment Technologies SA');
-        $this->setView('TaoEventLog/templateExample.tpl');
+    /**
+     * Load json data with results
+     */
+    public function search()
```

Comment:
All params will be checked in RdsStorage (they all are optional). It's controller for datatable component, so it uses always such way.